### ISSUE TITLE
[Bug] Cannot navigate via EventToCommandBehavior in XAML

### ISSUE THREAD

**Issue by Nate129** [PrismLibrary/Prism#2484]
### Description

When attempting to navigate via XAML and `EventToCommandBehavior` like so: 

``` c#
<ListView.Behaviors>
      <prism:EventToCommandBehavior EventName="ItemTapped"
                                    Command="{prism:NavigateTo 'DetailPage'}"
                                    EventArgsParameterPath="Item" />
</ListView.Behaviors>
```

The following exception is thrown:

```
System.ArgumentNullException: 'Value cannot be null.
Parameter name: TargetElement'
```

### Steps to Reproduce

1. Setup a ListView containing elements in XAML
2. Use `EventToCommandBehavior` to execute a command on the `ItemTapped` event
3. Use Prism XAML navigation in the Command property for the `EventToCommandBehavior`
4. The exception is thrown when tapping an item in the ListView

### Expected Behavior

The navigation to occur when tapping the item.

### Actual Behavior

The above exception is thrown.

### Basic Information

- Version with issue: 8.1.97
- Last known good version: n/a
- Xamarin.Forms version: 5.0.0.2012
- IDE: Visual Studio 16.9.6

### Reproduction Link

A Visual Studio solution that demonstrates the issue is [here](https://github.com/Nate129/PrismEventNavRepro). Run the app on Android, tap a name in the ListView, the exception will occur. 

Thanks!

**Comment by brianlagunas** [comment]
Thanks for reporting this. This was never designed to work within a Behavior. For now you will have to use an ICommand in your ViewModel. I will add support for calling the `NavigateTo` markup extension within the Prism `EventToCommand`.

**Comment by Nate129** [comment]
You're welcome! I'm glad to hear that the ability to navigate via an event will be supported in XAML without the need to add any extra code to the code behind or ViewModel. It will make the task of writing code to navigate via an event like `ItemTapped` on a `ListView` cleaner and more succinct.

**Comment by dansiegel** [comment]
closed by #2495
